types/netlify-identity-widget/index.d.ts

```diff
@@ -0,0 +1,88 @@
+// Type definitions for netlify-identity-widget 1.4
+// Project: https://github.com/netlify/netlify-identity-widget
+// Definitions by: Naveen Kumar Sangi <https://github.com/nkprince007>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export { };
```

Comment:
This line is unnecessary.
In order to have an ES6 style, callable import statement when consuming these type definitions, the consumer must enable `--esModuleInterop` (or more specifically, `--allowSyntheticDefaultImports`), thereby allowing them to use `import NetlifyIdentityWidget from "netlify-identity-widget";` in the same way that one would use `import NetlifyIdentityWidget = require("netlify-identity-widget");`